crates/nu-cmd-extra/src/extra/bits/and.rs

```diff
@@ -42,52 +58,82 @@ impl Command for BitsAnd {
         input: PipelineData,
     ) -> Result<PipelineData, ShellError> {
         let head = call.head;
-        let target: i64 = call.req(engine_state, stack, 0)?;
+        let target: Value = call.req(engine_state, stack, 0)?;
+        let endian = call.get_flag::<Value>(engine_state, stack, "endian")?;
```

Comment:
You can use `call.get_flag::<Spanned<String>>(engine_state, stack, "endian")`.  So you don't need to check `Value` variant